scripts/mktcli.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env babel-node
+
+import { eth, Log, cli, contracts } from 'decentraland-commons'
+import { loadEnv } from './utils'
+import { decodeAssetId, encodeAssetId } from './monitor/utils'
+
+const log = new Log('mktcli')
+
+const logError = err => log.error('ERR: ' + err.message)
+const parseCoords = coords =>
+  coords
+    .replace('(', '')
+    .replace(')', '')
+    .split(',')
+
+const landRegistry = () => eth.getContract('LANDRegistry')
+const marketplace = () => eth.getContract('Marketplace')
```

Comment:
Maybe this should be called `getMarketplace` or  `getMarketplaceContract` ? so we know that they're returning something